src/main/java/uk/gov/di/ipv/cri/common/library/persistence/item/CanonicalAddress.java

```diff
@@ -148,6 +141,14 @@ public void setAddressCountry(String addressCountry) {
         this.addressCountry = addressCountry;
     }
 
+    public String getAddressRegion() {
```

Comment:
I see this file doesn't have any tests but I wonder if there are others that use this class that could be updated?

Reply:
Not really within this repo. The only test class that `CanonicalAddress` gets used in is https://github.com/govuk-one-login/ipv-cri-lib/blob/main/src/test/java/uk/gov/di/ipv/cri/common/library/service/PersonIdentityMapperTest.java 

But if we are not adding addressRegion to the Address object and only CanonicalAddress then it would not actually test/assert anything for addressRegion.